src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends FigureColor {
```

Comment:
Square should not extend FigureColor. Consider using an interface to define behavior for color and a separate class to hold the color property.